README.md

```diff
@@ -1,192 +1,135 @@
+
+The `create-bosh-concourse` pipeline can be run on from Concourse Lite, which will create the persistent AWS resources, and configure the software running on top of them.
+
+The persistent resources created will include a production configuration of Concourse, with the `create-bosh-concourse` pipeline.
+
+## Deploying a new environment
+`paas-bootstrap` can build the foundations for two types of environments:
+* CloudFoundry
+* Build environments (i.e. docker containers, bosh releases etc)
+
+The full bootstrap process below is only needed the first time an environment is deployed.
+
+These instructions contain placeholders where the exact command may vary. The below table explains the purpose of those placeholders
+
+| Placeholder   | Purpose                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------|
+| `$ACCOUNT`    | The AWS account being targeted (e.g. `dev`, `staging`)|
+| `$ENV` | <p>The name of the environment being targeted. In the case of short lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars).</p> <p>Do not use the same `DEPLOY_ENV` value for both types of bootstraps, because that will cause resource allocation conflicts and things will break</p>|
+
+### Pre-requisites
+
+- [ ] Make
+- [ ] [GDS CLI](https://github.com/alphagov/gds-cli)
+- [ ] [jq](https://github.com/stedolan/jq)
+- [ ] Access to `paas-credentials` (private repository) and tools installed
+- [ ] Connection to the GDS VPN
+- [ ] Permission to assume the `Admin` role of the relevant AWS account (dev, ci, staging, production)
+- [ ] `AWS_DEFAULT_REGION` environment set the desired region for the environment
+- [ ] [Github OAuth application credentials](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
+  If you're deploying a production/staging/shared dev environment (e.g. dev01, dev02), the app should belong to the alphagov organisation, and the credentials should be added to `paas-credentials` <br />
+  If you're deploying a short-lived environment (e.g. for penetration testing), they can be associated with your GitHub account, and you can put them in your personal password store.
+  Set them as environment variables:
+  ```
+  export GITHUB_CLIENT_ID=some-github-client-id
+  export GITHUB_CLIENT_SECRET=some-github-client-secret
+  ```
+
+## Deploying the bootstrap for CloudFoundry
+
+1. Clone `paas-bootstrap` at the tip of the `main` branch
+2. Launch Concourse Lite.
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV deployer-concourse bootstrap
+    ```
+3. Log in to Concourse List at http://127.0.01:8080/login using the credentials shown in the output from step 2
```

Comment:
```suggestion
3. Log in to Concourse List at http://127.0.0.1:8080/login using the credentials shown in the output from step 2
```

Reply:
I think you have inadvertently re-introduced this typo 😆

Reply:
Good catch